lib/adapter/TingClientRequestAdapter.php

```diff
@@ -69,7 +69,7 @@ public function execute(TingClientRequest $request) {
         $stopTime = explode(' ', microtime());
         $time = floatval(($stopTime[1]+$stopTime[0]) - ($startTime[1]+$startTime[0]));
 
-        $this->logger->log('Completed SOAP request ' . $soapAction . ' ' . $request->getWsdlUrl() . ' (' . round($time, 3) . 's). Request body: ' . $client->requestBodyString . ' Response: ' . $response);
+        $this->logger->log('Completed SOAP request ' . $soapAction . ' ' . $request->getWsdlUrl() . ' (' . round($time, 3) . 's). Request body: ' . $client->requestBodyString );
```

Comment:
If it is not a part of the issue then it should not be a part of the pull request. Please remove it.

I agree that crashing watchdog is a problem, but then we should deal with the situation separately. Removing the response entirely also makes debugging harder. Another option would be to cap the response at an appropriate string length.